spec/spec_helper.rb

```diff
@@ -2,27 +2,28 @@
 require 'rspec-puppet-facts'
 include RspecPuppetFacts
 
-if Dir.exist?(File.expand_path('../../lib', __FILE__))
-  require 'coveralls'
-  require 'simplecov'
-  require 'simplecov-console'
-  SimpleCov.formatters = [
-    SimpleCov::Formatter::HTMLFormatter,
-    SimpleCov::Formatter::Console
-  ]
-  SimpleCov.start do
-    track_files 'lib/**/*.rb'
-    add_filter '/spec'
-    add_filter '/vendor'
-    add_filter '/.vendor'
-  end
-end
+# if Dir.exist?(File.expand_path('../../lib', __FILE__))
+#   require 'coveralls'
+#   require 'simplecov'
+#   require 'simplecov-console'
+#   SimpleCov.formatters = [
+#     SimpleCov::Formatter::HTMLFormatter,
+#     SimpleCov::Formatter::Console
+#   ]
+#   SimpleCov.start do
+#     track_files 'lib/**/*.rb'
+#     add_filter '/spec'
+#     add_filter '/vendor'
+#     add_filter '/.vendor'
+#   end
+# end
 
 RSpec.configure do |c|
   default_facts = {
     puppetversion: Puppet.version,
     facterversion: Facter.version
   }
+  c.backtrace_exclusion_patterns << /\.vendor/
```

Comment:
can you check the rubocop warning on this line?

Reply:
will also remove this afterwars. this is needed or i cant see tests because they are digged under backtrace stuff